server/routes/api/conversations.js

```diff
@@ -78,4 +77,19 @@ router.get("/", async (req, res, next) => {
   }
 });
 
-module.exports = router;
+
+router.post("/resetUnreadMsgForCurrentUser" ,async (req, res, next) => {
```

Comment:
There's a couple of things I want to mention about this route:

- Because we're updating a resource instead of creating one, the correct HTTP verb to use here would be `PATCH` or `PUT` instead of `POST`. You can read about the differences in [this SO post](https://stackoverflow.com/questions/31089221/what-is-the-difference-between-put-post-and-patch#:~:text=POST%20is%20always,update%20a%20resource).
- Usually, we want the names of the endpoints to be a noun that describes the resource it's updating instead of a verb. In this case it could be something like "read" or "read-status" (also note the use of "kebab-case" instead of "camelCase" in the route name). Read more about that [here](https://restfulapi.net/resource-naming/#:~:text=2.1.%20Use%20nouns%20to%20represent%20resources).